.sass-lint.yml

```diff
@@ -4,6 +4,7 @@ files:
   ignore:
     - 'client/src/styles/legacy/*'
     - 'admin/client/src/styles/legacy/*'
+    - 'src/Dev/**/*'
```

Comment:
Doesn't need to be fixed in this PR, but this is a surprising place to put javascript. `src/` is for PHP.